Composer/packages/client/src/pages/botProject/AppIdAndPassword.tsx

```diff
@@ -114,14 +103,27 @@ export const AppIdAndPassword: React.FC<AppIdAndPasswordProps> = (props) => {
   }, [projectId, mergedSettings, localMicrosoftAppId]);
 
   return (
-    <CollapsableWrapper title={formatMessage('App Id / Password')} titleStyle={title}>
+    <Fragment>
+      <div css={title}>{formatMessage('Microsoft App ID')}</div>
+      <div css={subtext}>
+        {formatMessage.rich(
+          'A Microsoft App ID is required for your local Azure resources. If you’ve created an App ID already, you can add here. If not, your App ID and secret will be created when you provision resources for this bot. <a>Learn more.</a>',
+          {
+            a: ({ children }) => (
+              <Link href={''} target="_blank">
```

Comment:
I'm assuming you're probably waiting on a link

Reply:
Yes adding it now